============================= test session starts ==============================
platform linux -- Python 3.11.8, pytest-8.0.0, pluggy-1.4.0
rootdir: /home/runner/work/Assessment-Backend-Week-1/Assessment-Backend-Week-1
collected 85 items

days_api/test_api.py ..........................................FFFFFFFFF [ 60%]
FFFFFFFF.                                                                [ 70%]
days_api/test_date_functions.py .........................                [100%]

======================== 17 failed, 68 passed in 0.23s =========================
************* Module app
days_api/app.py:7:0: W0611: Unused Response imported from flask (unused-import)
************* Module date_functions
days_api/date_functions.py:11:8: W0707: Consider explicitly re-raising using 'except ValueError as exc' and 'raise ValueError('Unable to convert value to datetime.') from exc' (raise-missing-from)

------------------------------------------------------------------
Your code has been rated at 9.90/10 (previous run: 9.90/10, +0.00)

